ovos_document_chunkers/text/sentence.py

```
@@ -27,19 +27,27 @@ def __init__(self, config: Optional[Dict] = None):
-                    yield s
-
+            if not c.strip():
+                continue
+            try:
+                for p in paragraph_tokenize(c):
+                    if not p.strip():
+                        continue
+                    try:
+                        for s in sentence_tokenize(p):
+                            yield s
+                    except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)